Makefile

```diff
@@ -131,7 +131,7 @@ binary: $(BINARY_NAME)  ## Build application binary (native)
 .PHONY: install
 install:  ## Install the CLI native binary into GOBIN
 	@echo "-- Installing native binary in $(shell go env GOBIN)"
-	go install $(GO_LDFLAGS) github.com/cldcvr/terrarium/src/cli/terrarium
+	CGO_ENABLED=1 go install $(GO_LDFLAGS) github.com/cldcvr/terrarium/src/cli/terrarium
```

Comment:
Now that I look at this, given that we moved all the cross-compile stuff out to a script/pipeline, I don't think we need the $(BINARY_NAME) target anymore? or the make_binary function? If we do, I think you will need to put the CGO_ENABLED=1 there as well....

Reply:
Done